src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/SentencingMappingResource.kt

```diff
@@ -58,6 +58,16 @@ class SentencingMappingResource(private val mappingService: SentencingMappingSer
         description = "Unauthorized to access this endpoint",
         content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
       ),
+      ApiResponse(
+        responseCode = "409",
+        description = "Indicates a duplicate sentencing adjustment mapping has been rejected. If Error code = 409 the body will return a DuplicateErrorResponse",
```

Comment:
seems odd this one is `409` when others are different?

Reply:
response code should be 409 and error code 1409

Reply:
I'll correct that error code that snuck in - should be 1409